tests/codes/eurocode/nen_en_1993_1_1_c2_a1_2016/chapter_6_ultimate_limit_state/test_formula_6_5.py

```diff
@@ -22,29 +23,28 @@ def test_evaluation(self) -> None:
 
         assert form == pytest.approx(expected)
 
-    def test_raise_error_when_negative_n_ed_is_given(self) -> None:
-        """Test a negative value for v_rd_s."""
-        # Example values
-        n_ed = -7
-        n_t_rd = 10
-
-        with pytest.raises(ValueError):
-            Form6Dot5UnityCheckTensileStrength(n_ed=n_ed, n_t_rd=n_t_rd)
-
-    def test_raise_error_when_negative_n__t_rd_is_given(self) -> None:
-        """Test a negative value for v_ccd."""
-        # Example values
-        n_ed = 7
-        n_t_rd = -10
-
-        with pytest.raises(ValueError):
-            Form6Dot5UnityCheckTensileStrength(n_ed=n_ed, n_t_rd=n_t_rd)
-
-    def test_raise_error_when_zero_n_t_rd_is_given(self) -> None:
-        """Test a negative value for v_td."""
-        # Example values
-        n_ed = 10
-        n_t_rd = 0
-
-        with pytest.raises(ValueError):
+    @pytest.mark.parametrize(
+        ("n_ed", "n_t_rd"),
+        [
+            (-7, 10),  # n_ed is negative
+            (7, -10),  # n_t_rd is negative
+        ],
+    )
+    def test_raise_error_when_negative_is_given(self, n_ed: float, n_t_rd: float) -> None:
+        """Test a negative value for n_ed or n_t_rd."""
+        with pytest.raises(NegativeValueError):
```

Comment:
@Simone-de-Rijke The reason it's failing it because n_trd wont give a NegativeValueError but a LessOrEqualToZeroError.
![afbeelding](https://github.com/Blueprints-org/blueprints/assets/44614728/5bd4c3a9-3517-4d6d-a33b-c7ca56a98ee7)

Reply:
you can solve it by adding multiple errors to the pytest.raises method, something like this:

`python        
with pytest.raises((LessOrEqualToZeroError,NegativeValueError )):
            Form6Dot5UnityCheckTensileStrength(n_ed=n_ed, n_t_rd=n_t_rd)
`

hope this will solve the issue

Reply:
Tests are now fixed